projects/storefrontlib/src/layout/breakpoint/breakpoint.service.spec.ts

```diff
@@ -282,40 +282,40 @@ describe('BreakpointService', () => {
         expect(isEqual(0, BREAKPOINT.xs)).toBeTruthy();
       });
 
-      it('should return true if current screen size equals xs', () => {
-        expect(isEqual(500, BREAKPOINT.xs)).toBeTruthy();
+      it('should return equals for xs on largest size', () => {
+        expect(isEqual(500-1, BREAKPOINT.xs)).toBeTruthy();
       });
 
       it('should return true if current screen size equals sm', () => {
-        expect(isEqual(700, BREAKPOINT.sm)).toBeTruthy();
+        expect(isEqual(700-1, BREAKPOINT.sm)).toBeTruthy();
       });
 
       it('should return true if current screen size equals md', () => {
-        expect(isEqual(900, BREAKPOINT.md)).toBeTruthy();
+        expect(isEqual(900-1, BREAKPOINT.md)).toBeTruthy();
       });
 
       it('should return true if current screen size equals lg', () => {
-        expect(isEqual(1200, BREAKPOINT.lg)).toBeTruthy();
+        expect(isEqual(1200-1, BREAKPOINT.lg)).toBeTruthy();
       });
 
       it('should return true if current screen size equals xl', () => {
-        expect(isEqual(1300, BREAKPOINT.xl)).toBeTruthy();
+        expect(isEqual(1300-1, BREAKPOINT.xl)).toBeTruthy();
```

Comment:
Just a comment but xl is min 1200 so 1300 should still work here

Reply:
:D copy / past, huh...